adoc/chapters/programming_interface.adoc

```diff
@@ -8472,6 +8501,16 @@ template parameter as [code]#image_target::host_task# and then use the
 [code]#unsampled_image_accessor# from a <<sycl-kernel-function>> are ill
 formed.
 
+An unsampled image accessor is only a view over its associated image.
+Therefore, it is the application's responsibility to ensure that the lifetime
+of the image is at least as long as the lifetime of the accessor.  Calling any
+member function on an unsampled image accessor after its associated image has
+been destroyed results in undefined behavior.  In some cases the image
+destructor is defined to block until commands with requisites on that image
+have completed.  This results in well defined behavior so long as all
+invocations to member functions of accessors for that image occur before those
+commands complete.
```

Comment:
Should we have more content about `image` lifetime and synchronization?
I have the feeling we only addressed buffers up to now in the specification.

Reply:
Your comment simplify the implementation, so I am fine with it.